storage/grpc_client.go

```diff
@@ -612,7 +612,11 @@ func (c *grpcStorageClient) DeleteObjectACL(ctx context.Context, bucket, object
 	return errMethodNotSupported
 }
 func (c *grpcStorageClient) ListObjectACLs(ctx context.Context, bucket, object string, opts ...storageOption) ([]ACLRule, error) {
-	return nil, errMethodNotSupported
+	o, err := c.GetObject(ctx, bucket, object, defaultGen, nil, nil, opts...)
```

Comment:
Is it important that we allow _any_ generation to be used? Or is the default generation adequate? I guess I don't see it used in the HTTP client.

Reply:
+1 to the ACLHandle currently does not support specifying a specific generation of an object. I've added a few comments that notes this case.